app/boards/arm/kbdfans_tofu65_v2/kbdfans_tofu65_v2.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: kbdfans_tofu65_v2
+name: KBDfans Tofu65 2.0
+type: mcu
+arch: arm
+flash: 8192
+ram: 264
```

Comment:
Check these.

Reply:
According to Raspberry Pi the RP2040 has "264kB on-chip SRAM in six independent banks"

Since there seems to be some consensus on the flash ram I'll up that to 16Mb.